src/identity/apis/auth.ts

```diff
@@ -283,3 +284,31 @@ export const updateDefaultOrgByAccountID = async ({
 
   // success status code is 204; no response body expected.
 }
+
+// fetch user default account's default org
+export const getDefaultAccountDefaultOrg = async (): Promise<OrganizationSummaries[number]> => {
+  const response = await getAccounts({})
+
+  if (response.status === 401) {
+    throw new UnauthorizedError(response.data.message)
+  }
+
+  if (response.status === 500) {
+    throw new ServerError(response.data.message)
+  }
+  const {data} = response
+
+  if (Array.isArray(data) && data.length) {
+    const defaultAccount = data.find(account => account.isDefault)
+
+    // fetch default org
+    if (defaultAccount) {
+      const quartzOrg = await fetchOrgsByAccountID(defaultAccount.id)
+      const defaultQuartzOrg =
+        quartzOrg.find(org => org.isDefault) || quartzOrg[0]
+      return defaultQuartzOrg
+    }
+  }
+
+  throw new GenericError('No default account found')
```

Comment:
I think I told you to write this so it's my mistake (listening to my adivce was your mistake 😉), but I think this should say `No default organization found`

Reply:
haha, I did copy paste your suggestion ...BUT i dont think you were wrong. If `if (Array.isArray(data) && data.length)` returns false we want to return an error that says something like `no account found` because it means we were able to fetch a response but the response had relevant data in it? Taking a second look at this is making me think maybe we should also wrap `const quartzOrg = await fetchOrgsByAccountID(defaultAccount.id)` in a `try{}catch{}` ... maybe something like this? 
<img width="604" alt="Screen Shot 2022-08-10 at 11 28 46 AM" src="https://user-images.githubusercontent.com/66275100/183963553-87d4b49d-47b4-49b2-b5ca-caf5b39bd18c.png">

Reply:
`fetchOrgsByAccountID` will throw an error on its own if it fails, which will bubble up to the code calling this function if it's not caught here, which means this function doesn't **need** to catch the error, but it can if it wants.

The `try/catch` here will catch an error thrown by `fetchOrgsByAccountID`, but it will still return `undefined` if `defaultQuartzOrg` comes back as `undefined` by the `find` function not finding anything, so this code will still need to account for that.